frontend/src/components/Prize/index.js

```diff
@@ -32,7 +33,7 @@ const Prize = ({ name }) => {
       </div>
 
       {name !== "nft" ? (
-        <Text>Visit the Subvisual booth to claim your prize.</Text>
+        <Text>Visit the {COMPANY} booth to claim your prize.</Text>
       ) : (
         <Text>Insert your address....</Text>
       )}
```

Comment:
Simplify this ternary. See frontend notes I sent you previously.

Reply:
It was simplified in Ricardo's PR so I chose not to fix it in the last commit I did. But I'll do it now